web/src/engine/websites/Akuma.ts

```diff
@@ -0,0 +1,47 @@
+import { Tags } from '../Tags';
+import icon from './Akuma.webp';
+import { Chapter, DecoratableMangaScraper, type Manga } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+
+const pageScript = `
+    new Promise( (resolve, reject) => {
+        try {
```

Comment:
The `try ... catch` is redundant in synchronous context scope.

Reply:
i use fetch.then, and its sync? gosh i will never understand